src/App.js

```diff
@@ -4,9 +4,61 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      Hello world!
+      <MessageElement></MessageElement>
+      <MessageComponent></MessageComponent>
+      <MessagePureComponent></MessagePureComponent>
+      <MessageFunctional></MessageFunctional>
     </div>
   );
 }
 
+/** Using functional component */
+function MessageFunctional() {
+  return (
+    <header className="App-message-container">
+      <h4 className="App-message-heading">Message created using <code>Functional component:</code></h4>
+      <span className="App-message">Hello World</span>
+    </header>
+  );
+}
+
+/** Using `React.Component` */
+class MessageComponent extends React.Component {
```

Comment:
let's get _Component/PureComponent/createElement_ from 'react' destructively

Reply:
I didn't get this point. Could you please elaborate it more.

Reply:
ok, for example: 
```
import React, { Component } from 'react';

class MessageComponent extends Component {....}
```